File: problemas/1000/1099.dart
import 'dart:io';
import 'dart:math';

void main() {
  final n = int.parse(stdin.readLineSync());

  for (var i = 0; i < n; i++) {
    List linha = stdin.readLineSync().split(' ');
    final x = int.parse(linha[0]);
    final y = int.parse(linha[1]);
    final menor = min(x, y);
    final maior = max(x, y);
    int soma = 0;

    for (var j = menor + 1; j < maior; j++) {
      if (j % 2 != 0) {
        soma += j;
      }
    }

    stdout.writeln(soma);
  }
}

File: problemas/1000/1099.js
const entrada = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n');
const n = Number(entrada.shift());

for (let i = 0; i < n; i++) {
  const [x, y] = entrada.shift().split(' ');
  const menor = Math.min(x, y);
  const maior = Math.max(x, y);
  let soma = 0;

  for (let i = menor + 1; i < maior; i++) {
    if (i % 2 !== 0) {
      soma += i;
    }
  }

  console.log(soma);
}